README.md

```diff
@@ -39,20 +39,40 @@ composer create-project ymcatwincities/openy-project MY_PROJECT --no-interaction
 
 This command will build project based on [**latest stable**](https://github.com/ymcatwincities/openy/releases) release.
 
-#### Latest DEVELOPMENT version
+#### Latest 2.0 BETA version
+```
+composer create-project ymcatwincities/openy-project:8.2.x-dev MY_PROJECT --no-interaction --no-dev
```

Comment:
```Installing drupal/core (8.4.0-alpha1): Loading from cache```
That's why we have https://github.com/ymcatwincities/openy#installation-of-open-y ```composer update``` command in a queue after composer create finished